editoast/src/views/timetable.rs

```diff
@@ -351,6 +359,29 @@ async fn conflicts(
     Ok(Json(conflict_detection_response.conflicts))
 }
 
+pub fn filter_core_work_schedule(
```

Comment:
Should this function be in an `impl WorkSchedule`? I'm really asking the question because you might have a better understanding than me if it makes sense or not.

Reply:
Done. (it was renamed by `map_to_core_work_schedule`)